high_card.rb

```diff
@@ -3,3 +3,30 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = []
+results = []
+
+suits.each do |suit|
+    ranks.each do |rank|
+      deck.push([rank,suit])
+    end
+end
+
+deck.shuffle!
+
+deck.each do |card|
+  puts card[0].to_s + " of " + card[1].to_s
+end
+
+loop do
+  puts "Enter players name to add to the game.  When all players are added type 'done' to play"
+  input = gets.chomp
+  if input == "play"
+    break
+  elsif players.push(input)
+  end
+end
+
+cards = players.map do |player|
+  deck.pop
+end
```

Comment:
Nice, now that we have a card for each player, we need to be able to compare the cards and declare a winner. One tricky part is once we have the value that is the highest, how do we map that back to a player?